src/imputation/knnimputer.py

```diff
@@ -20,7 +20,7 @@ def _execute(self) -> pd.DataFrame:
                     self.ml_data.dataset_descriptor.input_column,
                     self.ml_data.dataset_descriptor.target_column,
                 ]
-            ].to_numpy()
+            ].values
```

Comment:
```suggestion
            ].to_numpy()
```

Reply:
`to_numpy()` reminds that calling it has a cost and invite to use an intermediate variable to store its result. `.values` is hiding it. And pandas has deprecated it, bad API design.